csm_web/scheduler/email/email_utils.py

```diff
@@ -0,0 +1,242 @@
+import os
+import base64
+import html
+from email.message import EmailMessage
+
+from django.conf import settings
+
+from google.oauth2.credentials import Credentials
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+from .email_errors import EmailFormattingError, NoEmailError, EmailAuthError, email_error_handling
+
+PROFILE_URL = "https://scheduler.csmentors.org/profile"
+
+# Email error handling wrapper adds logger argument as last argument
+
+
+@email_error_handling
+def email_enroll(student):
+    """Sends the corresponding mentor an email notification that the STUDENT enrolled in their section"""
+
+    try:
+        mentor = student.section.mentor
+        course_title = student.course.title
+        mentor_name = mentor.user.first_name + " " + mentor.user.last_name
+        student_name = student.user.first_name + " " + student.user.last_name
```

Comment:
There is a `User.get_full_name()` method in the `User` model that can be used here, just to keep things more consistent.